wavelink/node.py

```diff
@@ -63,9 +64,9 @@
     )
     from .types.tracks import TrackPayload
 
-    LoadedResponse = Union[
-        TrackLoadedResponse, SearchLoadedResponse, PlaylistLoadedResponse, EmptyLoadedResponse, ErrorLoadedResponse
-    ]
+    LoadedResponse = (
+        TrackLoadedResponse | SearchLoadedResponse | PlaylistLoadedResponse | EmptyLoadedResponse | ErrorLoadedResponse
+    )
```

Comment:
Triple check this is valid in use. There has been some issues where the union type needs to be stringified to work correctly. 
E.g. `"A | B | C"`

Reply:
This should work since future annotations is imported, I think, but I can put `TypeAlias` in front of it just in case and try to run a simple bot, see if it breaks that way.